README.md

```diff
@@ -31,30 +31,31 @@ $ git submodule update
 ```
 ## Compilation:
 **1** Install 3d-parties developers libraries
+- Run the following commands :
 ```
-sudo apt-get install liblua5.2-dev libxml2-dev lua-lpeg-dev
-sudo apt-get install openssl
+$ sudo apt-get install liblua5.2-dev libxml2-dev lua-lpeg-dev
+$ sudo apt-get install openssl
 ```
 
-**2** Install Qt5.3+
-
-[Download Qt5.3+](https://download.qt.io/archive/qt/5.3/5.3.1/)
+**2** Install Qt5.9+
+- Run the following commands :
 ```
-sudo chmod +x some-qt-build.run
-./some-qt-build.run
+$ sudo add-apt-repository -y ppa:beineri/opt-qt591-xenial
```

Comment:
These steps do not work for ubuntu 18 Bionic

Reply:
@JackLivio Yes, you're correct. This update is applicable for `16.04`. For `18.04` the following should work:
```
qt5-default libqt5websockets5-dev
```

Reply:
@JackLivio Instruction has been updated accordingly (see 17251cf)